Dor

Dorelhayani opened the pull request:
up to date :
complete code with plant status , plant status history , Device Location pages . 
option to view each plant history from the main page,  each cell with status ( connected / disconnected )  will navigate to its history page. 
adding device location

Dorelhayani committed:
Json Table Create

Dorelhayani committed:
Json Table Create

Dorelhayani committed:
Displaying plant status on table

Dorelhayani committed:
added history ejs
added history js
sorting by sensor , name, date

Dorelhayani committed:
update js file

Dorelhayani committed:
adding data files to middleware and routs

Dorelhayani committed:
Updating HistoryPage

Dorelhayani committed:
fixing some xampp issues.
fetching routs/data
work on progress

Dorelhayani committed:
stam

Dorelhayani committed:
feature added
every cell with data
(connected / disconnected) is a link to it's history page.

cell with no data stays in the main page

Dorelhayani committed:
Updated Git Code with DVLP

Adding abolel branch features

bug fixing

adding Device Location js & ejs filse.

amird55 reviewed (changes requested):
אל תכניס את קבצי צד שרת לקומיט
אני אכניס ידנית את הקוד שלך ולכן אני לא מאשר את ה-PR

Dorelhayani committed:
adding a graph history page.

graph displays min, max and value of each plant sensor.

x axis: time stamp
y axis: value

Dorelhayani committed:
integration

updating plant_status.js
updating history_page.js

to be working with database.

Dorelhayani commented:
integration 

fixing demo page 
------------------------------------------------------------------------------------------
1. plant / cell with data  won't navigate by clicking it to its plant history's page 
2.  select data to display ( displaySelectedData() function ) won't display the data 
------------------------------------------------------------------------------------------


updating history page 
------------------------------------------------------------------------------------------
history page work with environmental-data-avg
------------------------------------------------------------------------------------------


updating history graph 
------------------------------------------------------------------------------------------
history graph work with environmental-data-avg
as well as fixing  graph curvature nonappearance 
------------------------------------------------------------------------------------------


comparing graph 
------------------------------------------------------------------------------------------
adding a new graph for displaying single sensor's value  for multi plants.
------------------------------------------------------------------------------------------
